src/System.Management.Automation/utils/Verbs.cs

```diff
@@ -592,6 +592,1018 @@ public static class VerbsOther
     }
 
     /// <summary>
+    /// Verb descriptions.
+    /// </summary>
+    public static class VerbDescriptions
```

Comment:
Descriptions needs to be in a resx file so they can be localized.

Reply:
Should it be internal?